pentaho/src/main/java/pt/webdetails/cpf/repository/pentaho/unified/UserContentRepositoryAccess.java

```diff
@@ -23,10 +23,9 @@
 import pt.webdetails.cpf.Util;
 import pt.webdetails.cpf.api.IUserContentAccessExtended;
 import pt.webdetails.cpf.repository.api.FileAccess;
-import pt.webdetails.cpf.repository.api.IUserContentAccess;
 
 /**
- * {@link IUserContentAccess} implementation for {@link IUnifiedRepository}
+ * IUserContentAccess implementation for IUnifiedRepository
```

Comment:
Why remove the links?

Reply:
Build errors related to IUnifiedRepository never being imported, will change to the correct implemented class `{@link IUnifiedRepositoryExtended}`.